src/w2grid.js

```diff
@@ -2217,7 +2217,11 @@
             // after event
             this.trigger($.extend(eventData, { phase: 'after' }));
             // do not refresh if loading on infinite scroll
-            if (this.last.xhr_offset == 0) this.refresh(); else this.scroll();
+            if (this.last.xhr_offset == 0) this.refresh(); 
```

Comment:
sounds good. but please if you use else {} then add {} for if too. I will do it this time.